### PR TITLE
[NO-TICKET] Rename web component stories

### PR DESCRIPTION
**Pull request by pwolfert**
## Summary

Rename the web component stories to match the element name and diffenentiate from the React components.

## Checklist

- [x] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[WNMGDS-####] Title` or [NO-TICKET] if this is unticketed work.
- [x] Selected appropriate `Type` (only one) label for this PR, if it is a breaking change, label should only be `Type: Breaking`
- [x] Selected appropriate `Impacts`, multiple can be selected.
- [x] Selected appropriate release milestone

### CODE REVIEW

File: packages/docs/src/images/house-large.pxd
Comment (zarahzachz) [inline packages/docs/src/images/house-large.pxd:1]: what is this?

File: tests/browser/snapshots/stories/web-components-ds-accordion-item--default-cmsgov--Mobile-Chrome.png
Comment (zarahzachz) [inline tests/browser/snapshots/stories/web-components-ds-accordion-item--default-cmsgov--Mobile-Chrome.png:1]: i thought [accordion items were excluded](https://github.com/CMSgov/design-system/blob/dfde3989dcd61d9c7991608cbed8ce606cf6aa82/tests/browser/stories.test.ts#L28)?

File: packages/docs/src/images/house-large.pxd
Comment (pwolfert) [reply packages/docs/src/images/house-large.pxd:1]: Thanks for catching it. I guess the last place I export an image to from my image editor was that folder for something on the doc site, and I didn't realize it was saving a file there by default yesterday when I was editing an image for a friend. It's a Pixelmator Pro file. Removing

File: tests/browser/snapshots/stories/web-components-ds-accordion-item--default-cmsgov--Mobile-Chrome.png
Comment (pwolfert) [reply tests/browser/snapshots/stories/web-components-ds-accordion-item--default-cmsgov--Mobile-Chrome.png:1]: Ohh, that's where they came from. At some point I got extra images from running the VRTs, and I was wondering how that happened. I'll update that exlusion string and remove the files.

**Review by zarahzachz — APPROVED** [review]

**Pull request merged**